packages/scripts/docs/audit.md

```diff
@@ -6,11 +6,17 @@ Performs static dependency analysis
 
 The following checks are performed when this command is invoked.
 
-- ## audit-ci -h
+- ## audit-ci --config <.audit-ci.json path>
 
   ### Purpose
 
-  This tool will check for CVEs on any dependencies of the project. Any dependency with a **high** or **critical** level CVE will emit an error.
+  This tool will check for CVEs on any dependencies of the project. Any dependency with a **high** or **critical** level CVE that have not been [whitelisted](https://github.com/IBM/audit-ci#options) will emit an error.
+
+  When possible, use the following whitelist priority to minimize whitelist surface area:
+
+  1. `path-whitelist`
```

Comment:
Whitelist here at this line because of the audit-ci's interface; we use line-exclusions on these instances e.g. `<!--alex ignore whitelist-->`

I don't want to use the term whitelist for anything other than a cli parameter or config option that cannot be renamed.

Reply:
Agree